### ISSUE TITLE
Improve display of unhandled CIMError exception

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#1350]
If client code gets a CIMError exception that it does not handle, the standard Python traceback simply displays the exception object, which currently results in something like this:

```
File "/usr/lib/python2.6/site-packages/pywbem/cim_operations.py", line 686, in imethodcall
raise CIMError(code, tup_tree[1]['DESCRIPTION'])
pywbem.cim_operations.CIMError: (7, u'')
```

This is a real life example of a WBEM server that did not return a DESCRIPTION element, but just the CIM status code. The resulting display is pretty cryptic and could be improved to show the CIM status code name, e.g.:

```
pywbem.cim_operations.CIMError: (7 (CIM_ERR_NOT_SUPPORTED), u'')
```

See issue #1330 for background and reference to real life.

**Comment by andy-maier** [comment]
Investigation of the behavior of displaying unhandled ex ceptions in a traceback reveals that the str() representation of the exception object is used:

With pywbem master (pre 0.13.0):

```
$ python -c "import pywbem; raise pywbem.CIMError(7,'bla')"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
pywbem.exceptions.CIMError: 7: bla, Connection id: None

$ python -c "import pywbem; print('%s' % pywbem.CIMError(7,'bla'))"
7: bla, Connection id: None

$ python -c "import pywbem; print('%r' % pywbem.CIMError(7,'bla'))"
CIMError(7, 'bla')
```

With pywbem 0.8:

```
$ python -c "import pywbem; raise pywbem.CIMError(7,'bla')"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
pywbem.cim_operations.CIMError: (7, 'bla')

$ python -c "import pywbem; print('%s' % pywbem.CIMError(7,'bla'))"
(7, 'bla')

$ python -c "import pywbem; print('%r' % pywbem.CIMError(7,'bla'))"
CIMError(7, 'bla')
```

**Comment by andy-maier** [comment]
Has been addressed in PR #1351.
